content/tracing/advanced_usage/logging.md

```diff
@@ -57,6 +57,9 @@ Logback XML Pattern:
 
 {{% /tab %}}
 {{% tab "Python" %}}
+from ddtrace import helpers
```

Comment:
Add an incipit such as: "The Python tracer exposes an helper to extract trace and span identifiers, as follows:"